src/librbd/cache/Utils.h

```diff
@@ -13,7 +13,7 @@ namespace util {
 
 template <typename T>
 bool is_pwl_enabled(T& image_ctx) {
-#if defined(WITH_RBD_RWL)
+#if defined(WITH_RBD_RWL) || defined(WITH_RBD_SSD_CACHE)
   return image_ctx.config.template get_val<bool>("rbd_rwl_enabled");
```

Comment:
Nit: probably we should split out the `rbd_rwl_enabled` config option to an enum `rbd_persistent_cache_mode` (`disabled`, `rwl`, or `ssd`)

Reply:
Done